scripts/install/install-risingwave.sh

```diff
@@ -48,11 +48,11 @@ fi
 
 ############# BREW INSTALL
 if [ "${USE_BREW}" -eq 1 ]; then
-  echo "Installing RisingWave@${VERSION} using Homebrew."
+  echo "Installing RisingWave using Homebrew."
   brew tap risingwavelabs/risingwave
   brew install risingwave
   echo
-  echo "Successfully installed RisingWave@${VERSION} using Homebrew."
+  echo "Successfully installed RisingWave using Homebrew."
```

Comment:
The version might mismatch. Latest homebrew version is not always available at the same time as the github release. So just remove it.